source/Xunit.Gherkin.Quick.ProjectConsumer/Addition/AddNumbers.feature

```diff
@@ -0,0 +1,29 @@
+﻿Feature: Add Numbers
```

Comment:
Same goes for the feature file and name too. Let's rename to conform to what it does.